docs/src/explanations/memories.md

```diff
@@ -110,7 +115,7 @@ class RWSmem extends Module {
 
 Here is an example single read/write port waveform, with [masks](#masks) (again, generated signal names and number of arrays may differ):
 
-![read/write ports example waveform](https://svg.wavedrom.com/github/freechipsproject/www.chisel-lang.org/master/docs/src/main/tut/chisel3/memories_waveforms/smem_rw.json)
+![read/write ports example waveform](https://svg.wavedrom.com/github/freechipsproject/www.chisel-lang.org/master/docs/src/main/resources/json/smem_rw.json)
```

Comment:
@debs-sifive can you 👁️ this? I think this is your drawing? Is this new link correct?

Reply:
I just fixed the original link that didn't exist. Let me know if the drawing changed.

Reply:
yep, looks right

Reply:
Nice! Thanks @debs-sifive